compose_to_rancher/__init__.py

```diff
@@ -6,20 +6,34 @@
 default_file_path = 'docker-compose.yml'
 default_output_file_path = 'docker-compose-v1.yml'
 
+EXCLUDED_OPTIONS = ['build', 'depends_on']
+
+#######################
+# Base Functions
+#######################
 
 def convert_compose_to_rancher(compose_v2_data):
     """
     Convert a parsed yaml data structure docker-compose V2 to a \
     V1 docker-compose data structure
     """
     new_compose = {}
-    for service, contents in compose_v2_data['services'].items():
+    for service, service_options in compose_v2_data['services'].items():
+        #error if no image option is declared
+        if 'image' not in service_options:
+            raise ValueError('no image in service {}'.format(option))
```

Comment:
I agree, I like the change, I referred to exactly that line (24), the `... .format(option)` should be `... .format(service)`. Otherwise I think it will throw an error about an undeclared variable `option`.